src/common.jl

```diff
@@ -17,6 +17,41 @@ export fromroots,
        isintegral,
        ismonic
 
+function lejaorder!(roots) # see https://doi.org/10.1023/A:1025555803588
+    if length(roots) <= 2
+        return roots
+    end
+    #ii = argmax(j -> abs(roots[j]), eachindex(roots))
+    ii = firstindex(roots)
+    rmax = abs(roots[ii])
+    for j in eachindex(roots)
+        rabs = abs(roots[j])
+        if rabs > rmax
+            ii = j
+            rmax = rabs
+        end
+    end
```

Comment:
Alternatively, Compat.jl provides that `argmax` method. Would depending on Compat be an option or should we keep this loop? (Same below.)

Reply:
I don't mind having some small copy of package code in this package for easier compatability. I'd keep the loop unless you have a strong preference.